test/add_test.js

```diff
@@ -65,6 +65,7 @@ calculateMD5(file, (err, md5) => {
     `       "file": "pdfs/${pdf}.pdf",\n` +
     `       "md5": "${md5}",\n` +
     '       "rounds": 1,\n' +
+    '       "scale": 1,\n' +
```

Comment:
Please remove this line, since this parameter should be optional and consequently only needs to be set then *not* equal to one (which isn't going to be that common).

By doing this you'll unnecessarily inflate the size of the `test_manifest.json` file, and I'd view this new parameter similar to the exiting (and optional) `firstPage`/`lastPage` parameters that we only set when actually needed.